specification/quota/resource-manager/Microsoft.Quota/preview/2023-06-01-preview/groupquota.json

```diff
@@ -1063,7 +1079,7 @@
               }
             },
             "schema": {
-              "$ref": "#/definitions/SubscriptionQuotaAllocations"
+              "$ref": "#/definitions/LROResponse"
```

Comment:
>LROResponse [](http://example.com/codeflow?start=37&length=11)

Why doesn't it return GroupQuotasEntity and provisioningState?

Reply:
The LROResponse has provisioningState